maven-plugin/src/test/java/io/jenkins/tools/incrementals/maven/DetectIndentTest.java

```diff
@@ -1,17 +1,14 @@
 package io.jenkins.tools.incrementals.maven;
 
+import org.junit.jupiter.api.Test;
+
 import java.util.Arrays;
 import java.util.Collection;
 
-import org.junit.Assert;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
-@RunWith(Parameterized.class)
 public class DetectIndentTest {
 
-  @Parameterized.Parameters
   public static Collection<Object[]> data() {
```

Comment:
Unused & didn't test anything within the past 4 years.

Reply:
Oh right, I disabled it completely 🤦 

Passes locally now:
![Screenshot 2023-05-22 at 20 05 15](https://github.com/jenkinsci/incrementals-tools/assets/13383509/12e238a0-82f6-43db-b4fc-621dc26a38a4)